static/solo-cta/index.js

```diff
@@ -280,21 +279,52 @@ function handleModalInitialized(e){
           runTime: clickData.runTime
       }, '*');
   }
-
-  if (utils.isIOS) {
-    var onOrientationChange = function () {
-        if (iframeModal && iframeModal.contentWindow) {
-            var width = iframeModal.parentNode.offsetWidth;
-            iframeModal.contentWindow.window.postMessage({
-                event: config.eventPrefix + ':modal_holder_resize',
-                size: [width, Math.floor(width * (9 / 16))]
-            },'*');
+  var detectOrientation = function(){
+      var orientation ="noDetected";
+      if(window.innerHeight > window.innerWidth){
+          orientation = "portrait"
+      }
+      if(window.innerWidth > window.innerHeight){
+          orientation = "landscape"
+      }
+      return orientation;
+  };
+  var fixPadding = function(){
+      var orientation = detectOrientation();
+      if(orientation === "landscape"){
+        if(config.hasOwnProperty("iframe_modal_body_padding")){
+          var paddingValue = config["iframe_modal_body_padding"];
+          paddingValue = paddingValue.split(" ");
+          if(paddingValue.length>0){
+            var clearValue = paddingValue[0].replace("px","");
+            iframeModal.contentWindow.document.querySelector("body").style.paddingTop = (parseInt(clearValue)+4)+"px"; 
+          }  
         }
+      }else{
+        iframeModal.contentWindow.document.querySelector("body").style.paddingTop = null;
+      }
+  };
+  if (utils.isMobile) {
+    var onOrientationChange = function () {
+        var round = 0;
+        setInterval(function(){
+          if(round===3) return;
+          round = round+1;
+          if (iframeModal && iframeModal.contentWindow) {
+              fixPadding();
+              var width = iframeModal.parentNode.offsetWidth;
+              iframeModal.contentWindow.window.postMessage({
+                  event: config.eventPrefix + ':modal_holder_resize',
+                  size: [width, Math.floor(width * (9 / 16))]
+              },'*');
+          }
+        },500);
     };
     var orientationChangeEvent = 'onorientationchange' in window ? 'orientationchange' : 'resize';
     window.removeEventListener(orientationChangeEvent,onOrientationChange, false);
     window.addEventListener(orientationChangeEvent,onOrientationChange, false);
```

Comment:
The function `onOrientationChange` is going to be called every time there's an orientation change.
The function uses a `setInterval` function that will run each half of a second and it won't be stopped at any time.
Take a look at https://jsfiddle.net/5oxLrqg5/1/

Please remove code, we need to figure a better way to fix this issue